.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description: Request a missing feature
+title: '[Feature] '
+labels: ['feature', 'needs triage']
+body:
+  - type: markdown
+    id: intro-md
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request! Please ensure you provide as much information as possible so we can better understand what you want us to add and so we can come up with the best solution for everyone.
+  - type: checkboxes
+    attributes:
+      label: '💻'
+      description: (Optional) Please check this box if you’re willing to open a PR to implement this feature. We’ll help you get started and answer any questions you have along the way :)
+      options:
+        - label: Would you like to implement this feature?
+  - type: checkboxes
+    id: existing-issue
+    attributes:
+      label: 'Verified feature request does not already exist?'
+      description: 'Please search to see if an issue already exists for the feature you have requested.'
+      options:
+        - label: 'I have searched and found no existing issue'
+          required: true
+    validations:
+      required: true
+  - type: textarea
+    id: pitch
+    attributes:
+      label: 'Pitch: what problem are you trying to solve?'
+      description: Please concisely describe the problem that motivated your feature request.
```

Comment:
Please describe in as much detail, the problem that motivated you to submit a feature request.